modules/administration/pages/monitoring.adoc

```diff
@@ -4,6 +4,13 @@
 You can monitor your {productname} environment using Prometheus and Grafana.
 {productname} Server and Proxy are able to provide self-health metrics, or install and manage a limited number of Prometheus exporters on Salt clients.
 
+[NOTE]
+====
+Prometheus fetches metrics using a pull mechanism, so it must be able to establish network communications with monitored clients.
```

Comment:
We should mention 'to' monitored clients instead of 'with'. The direction of the connection goes from the Prometheus server *to* the client

Reply:
@cavalheiro Fixed.